apps/studio/src/features/editing-experience/components/LinkEditor/utils.ts

```diff
@@ -1,23 +1,32 @@
-export const getLinkHrefType = (href: string | undefined) => {
+import {
+  LINK_TYPE_EMAIL,
+  LINK_TYPE_EXTERNAL,
+  LINK_TYPE_FILE,
+  LINK_TYPE_PAGE,
```

Comment:
issue: we should make use of a JavaScript object instead to define the link types. For example:

```ts
export const LINK_TYPE = {
    Email: "Email",
    External: "External",
    File: "File",
    Page: "Page"
}
```

That way, we can use things like `LINK_TYPE.Email` to represent an email link, and to perform exhaustiveness checks.